apps/src/gamelab/GameLabVisualizationColumn.jsx

```diff
@@ -81,7 +81,7 @@ var GameLabVisualizationColumn = React.createClass({
   renderGridCheckbox() {
     return (
       <div onClick={() => this.props.toggleShowGrid(!this.props.showGrid)}>
-        <i id="grid-checkbox" className="fa fa-square-o"/>
+        <i id="grid-checkbox" className="fa fa-square-o" style={{width: 14}} />
```

Comment:
This could also be fixed by adding the special class `fa-fw` which makes FontAwesome use its fixed-width variant.